packages/authentication/package.json

```diff
@@ -59,7 +59,9 @@
     "@mikro-orm/postgresql": "5.9.7",
     "awilix": "^8.0.0",
     "dotenv": "^16.1.4",
+    "jsonwebtoken": "^9.0.2",
     "knex": "2.4.2",
+    "oauth": "^0.10.0",
```

Comment:
comment: should we have a look at the. OAuth2 packages and what is the diff between this one handling it and the proper oauth 2 package